test/tests/transfers/check_postprocessor_execute_on/tests

```diff
@@ -0,0 +1,44 @@
+[Tests]
```

Comment:
there's no official rule about the format but this needs reformatting to look like other tests spec